File: C++/598. Range Addition II.cpp
// Time: O(len)
class Solution {
public:
    int maxCount(int m, int n, vector<vector<int>>& ops) {
        int len = ops.size();
        for(int i=0;i<len;i++)
        {
            m = min(m, ops[i][0]);
            n = min(n, ops[i][1]);
        }
        return n*m;
    }
};

File: Python/598. Range Addition II.py
# Time: O(len)
class Solution:
    def maxCount(self, m, n, ops):
        """
        :type m: int
        :type n: int
        :type ops: List[List[int]]
        :rtype: int
        """
        for i in range(len(ops)):
            m = min(m, ops[i][0])
            n = min(n, ops[i][1])
        return n*m
